solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -179,11 +181,14 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         // sync the reserve balances
         syncReserveBalance(_sourceToken);
 
-        // calculate protocol fee from the conversion fee only. So it would take x% of conversion fee as the protocol fee.
-        // Also we don't need to substract anything from the return amount.
-        uint256 calculatedProtocolFee = calculateProtocolFee(_targetToken, fee);
+        /// @dev Protocol fee is just part of the fee above.
+        uint256 calculatedProtocolFee = calculateProtocolFee(fee);
+
+        /// Add the total helds token fee.

```

Comment:
This sentence makes little sense in english and is not very explicit either.
Suggestion "Add this transaction's protocol fee amount to the total fee amount held in the contract."